### PR TITLE
FE powers details page

### PR DESCRIPTION
**Pull request by kalidiagne**
- [x] Powers details page for `Telegram`,`QRCode`, `Twitter`
- [x] Optmize `useQuery` to fetch cached endpoints for `cards list` 

@AndrewCLu I need an overview for the `powers` endpoints in order to close also this part with all the details.

 Twitter Power

 Telegram Power

 QRCode Power

### CODE REVIEW

File: apps/jubmoji-quest/src/hooks/useFetchCards.ts [hunk 1]
```diff
@@ -29,6 +30,34 @@ export const useFetchCards = () => {
...
+          return getJubmojiCardByPubIndex(jubmojiCollectionCards, pubKeyIndex);
+        }) ?? [];
+      return collectedJubmojis;
+    },
+    {
+      enabled: !isLoading && !isLoadingJubmojis,
+    }
+  );
+};
+
```
Comment (kalidiagne) [inline apps/jubmoji-quest/src/hooks/useFetchCards.ts:60]: @vb7401 @AndrewCLu we can use this `useFetchCollectedCards ` whenever we need the list of the collected cards

File: apps/jubmoji-quest/src/hooks/useFetchCards.ts [hunk 2]
```diff
@@ -29,6 +30,34 @@ export const useFetchCards = () => {
...
+ * Get list of the collected cards with all the information
+ * @returns JubmojiCardProps
+ */
+export const useFetchCollectedCards = () => {
+  const { isLoading, data: jubmojiCollectionCards = [] } = useFetchCards();
+  const { isLoading: isLoadingJubmojis, data: jubmojis = [] } = useJubmojis();
+
+  return useQuery(
+    ["collectedCards", jubmojis?.length],
+    async () => {
```
Comment (AndrewCLu) [inline apps/jubmoji-quest/src/hooks/useFetchCards.ts:43]: Can we get a type here for readability?

**Review by vb7401 — APPROVED** [review]
LGTM!

**Comment by kalidiagne** [comment]
🚀

**Pull request merged**